src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -97,6 +98,8 @@
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">0</MaxRetryCount>
     <ToolsDir Condition="'$(ToolsDir)' == ''">$(MSBuildThisFileDirectory)</ToolsDir>
     <IsDropPublic Condition="'$(IsDropPublic)' == ''">false</IsDropPublic>
+    <!-- Make sure HelixBlobPrefix ends with '/' -->
+    <HelixBlobPrefix Condition="'$(HelixBlobPrefix)' != ''">$([System.Text.RegularExpressions.Regex]::Replace('$([MSBuild]::EnsureTrailingSlash('$(HelixBlobPrefix)'))', "\\", "/"))</HelixBlobPrefix>
```

Comment:
Does [MSBuild]::EnsureTrailingSlash() work on Desktop MSbuild?  I suspect this would break usage from there.

Reply:
Poked around, this works on Dev15 desktop MSBuild, so not a blocker but should probably call out this wouldn't work on Dev14.

Reply:
I'll add a comment saying as much

Reply:
msbuild already has a function for this called HasTrailingSlash see https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2017.

Reply:
Still not sure why HasTrailingSlash built-in function doesn't work for this scenario.

Reply:
`HasTrailingSlash` would work there as well, but require 2 lines.  It's possible to do it all in one line by using the `EnsureTrailingSlash` method.  Is there a version support concern that makes this change worth amending the PR?

Reply:
Not a version support concern just a readability concern. 

You can see the common pattern in the sdk itself https://github.com/dotnet/sdk/blob/ab2176b3447e3e7820b5a340408110b6d498f907/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DefaultOutputPaths.targets#L37